### DISCUSSION TITLE
Welcome to arv32-opt Discussions!

### DISCUSSION THREAD

**Discussion by raspiduino** [raspiduino/arv32-opt#2] [Announcements]
Hello everyone!

**Comment by raspiduino** [comment]
![Screenshot_20231002-163529_Chrome](https://github.com/raspiduino/arv32-opt/assets/68118236/a6933403-8515-4a40-91f6-dba40e65b017)
Nice. Thank you all!

**Reply by Yrkagannoo** [reply]
Hi

**Comment by ElectroBoy404NotFound** [comment]
It is at 94 stars now 👀

**Reply by raspiduino** [reply]
92
